drivers/exec/driver_test.go

```diff
@@ -102,7 +102,7 @@ func TestExecDriver_StartWait(t *testing.T) {
 	}
 
 	tc := &TaskConfig{
-		Command: "cat",
+		Command: "echo",
```

Comment:
this test changed because cat is not inside the chroot in the test suite. In the upcoming patch I added cat to the chroot instead of changing this test